azkaban-common/src/main/java/azkaban/executor/ExecutableFlow.java

```diff
@@ -68,6 +68,17 @@
   private ExecutableFlowRampMetadata executableFlowRampMetadata;
   private String flowLockErrorMessage;
 
+  // added for CloudFlow
+  public static final String EXPERIMENTID_PARAM = "experimentId";
+  public static final String DESCRIPTION_PARAM = "description";
+  public static final String FLOWDEFINITIONID_PARAM = "flowDefinitionId";
+  public static final String FLOWVERSION_PARAM = "flowVersion";
```

Comment:
Nit: please add underscores at word boundaries `EXPERIMENT_ID_PARAM`  `FLOW_VERSION_PARAM` etc.

Reply:
Following the naming conventions in ExecutableFlow file.